pkg/inference/scheduling/loader.go

```diff
@@ -209,7 +209,11 @@ func (l *loader) Unload(ctx context.Context, unload UnloadRequest) int {
 		if unload.All {
 			return l.evict(false)
 		} else {
-			return l.evictRunner(unload.Backend, unload.Model)
+			evicted := 0
+			for _, model := range unload.Models {
+				evicted += l.evictRunner(unload.Backend, model)
```

Comment:
```suggestion
				evicted += len(l.runners) - l.evictRunner(unload.Backend, model)
```
The result of `evictRunner` is the number of models remaining after eviction.  I don't know if I love this solution, but I think it's technically correct.

Reply:
I think the better thing to do would actually just be to not compute `evicted` here and just return `len(l.runners)` in the `else` case of the anonymous `func() int`.

Reply:
The `Unload` method does return the number evicted (there's a `len(l.runners) - func() int` computation happening).

Reply:
🤦 true, I don't know how I managed to miss that